src/versionUtils.ts

```diff
@@ -134,23 +119,18 @@ export async function getLatestCamelKVersion(): Promise<Errorable<string>> {
 	}
 }
 
-function checkKamelCLIVersion(): Promise<string> {
-	return new Promise<string>(async (resolve, reject) => {
-		const kamelLocal = kamelCli.create();
-		await kamelLocal.invoke('version')
-			.then((rtnValue) => {
-				const strArray = rtnValue.split(' ');
-				const version = strArray[strArray.length - 1].trim();
-				console.log(`Apache Camel K CLI (kamel) version returned: ${version}`);
-				resolve(version);
-				return;
-			}).catch(kamelUnavailableRejection(reject));
-	});
+async function checkKamelCLIVersion(): Promise<string> {
```

Comment:
I'm a little concerned about this refactoring and the lack of a catch if the version doesn't exist. I tested it locally and got some funky errors in the console, but nothing conclusive. Can we add a try/catch around it?

Reply:
A possible exception is catched in `checkKamelNeedsUpdate` function which should imo make a try catch inside this function obsolete.

Reply:
I changed the code a bit to try/catch directly in this function and removed it from the calling one.